### PR TITLE
chore (refs T34967): Add DpVideoPlayer to Storybook.

### PR DESCRIPTION
**Pull request by ahmad-demos**
https://yaits.demos-deutschland.de/T34967

- Add DpVideoPlayer to Storybook.

### CODE REVIEW

File: src/components/core/DpVideoPlayer.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    sources: object[]
+}
+
+type Story = StoryObj<IDpVideoPlayer>
+
+export default meta
+
+export const Default: Story = {
+    args: {
+        iconUrl: 'videoIconUrl.svg',
```
Comment (spiess-demos) [inline src/components/core/DpVideoPlayer.stories.tsx:30]: If you provide an empty string, the plyr svg is actually found.

```suggestion
        iconUrl: '',
```

Maybe you could add this link to the docblock of the prop within DpVideoPlayer: https://github.com/sampotts/plyr#using-the-iconurl-option

File: src/components/core/DpVideoPlayer.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        id: 'videoId',
+        sources: [{
+            src: 'videoSrc',
+            type: 'videoType'
```
Comment (spiess-demos) [inline src/components/core/DpVideoPlayer.stories.tsx:34]:
```suggestion
            type: 'mimeType'
```

File: src/components/core/DpVideoPlayer.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+The VideoPlayer component used to add Video Playback and can include various features and functionalities, such as:
+
+`Video Playback, Controls, Full-Screen Mode and more...`
+
+## Default usage
+
+```html
+<dp-video-player
+    :sources="[ { src: 'pathUrl', type: 'videoType' } ]"
```
Comment (spiess-demos) [inline src/components/core/DpVideoPlayer.stories.mdx:19]:
```suggestion
    :sources="[ { src: 'pathUrl', type: 'mimeType' } ]"
```

File: src/components/core/DpVideoPlayer.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+<Meta
+    title="Components/VideoPlayer"
+    component={DpVideoPlayer}
+/>
+
+# VideoPlayer
+
+The VideoPlayer component used to add Video Playback and can include various features and functionalities, such as:
+
+`Video Playback, Controls, Full-Screen Mode and more...`
```
Comment (spiess-demos) [inline src/components/core/DpVideoPlayer.stories.mdx:13]:
```suggestion
The VideoPlayer component is used to add Video content. It includes controls for play/pause, sound volume, playback speed, a download link, and a fullscreen view.
```

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: src/components/core/DpVideoPlayer.stories.tsx (hunk 1 above)
Comment (ahmad-demos) [reply src/components/core/DpVideoPlayer.stories.tsx]: Done in https://github.com/demos-europe/demosplan-ui/pull/590/commits/bb799bc52a78bff443895392a9d3f92762e43fd8

**Review by spiess-demos — APPROVED** [review]
lgtm!

**Pull request merged**